src/puzzle/sass/_app.scss

```
@@ -0,0 +1,84 @@
+@charset 'UTF-8';
+
+@use 'variable';
+@use 'mixin';
+
+header {
+  position: relative;
+  height: 100px;
+  width: 400px;
+  background-color: variable.$navy;
+
+  ul {
+    position: absolute;
+    right: 20px;
```

position の絶対指定はなるべく避けるようにして、flex などを使用して並べるようにしましょう。
絶対指定（`right: 20px` など）をしてしまうと、スマホサイズでは `right: 10px` にして…このサイズでは何pxにして…となってしまいます。

なので、flex 等を使用すると整列してくれるので、画面幅の影響を受けにくくなります。

他の箇所も同様なので修正をお願いします。

```suggestion
header {
  height: 100px;
  width: 400px;
  background-color: variable.$navy;

  ul {
```

---

修正しました。